Sources/MoSoClient/Endpoint.swift

```diff
@@ -14,17 +14,12 @@ struct RecommendationsEndpoint: Endpoint {
     private let queryItems: [URLQueryItem]
 
     init(recommendationCount: Int? = nil) throws {
-        guard let info = Bundle.main.infoDictionary, let key = info[Constants.consumerKeyKey] as? String else {
-            throw MoSoClientError.consumerKeyNotFound
-        }
         var actualCount = Constants.defaultRecommendationsCount
         if let recommendationCount, recommendationCount >= 0, recommendationCount <= Constants.defaultRecommendationsCount {
             actualCount = recommendationCount
         }
         self.queryItems = [
-            URLQueryItem(name: Constants.consumerKey, value: key),
-            URLQueryItem(name: Constants.locale, value: Locale.current.language.minimalIdentifier),
-            URLQueryItem(name: Constants.region, value: Locale.current.region?.identifier),
+            URLQueryItem(name: Constants.locale, value: Locale.current.identifier(.bcp47)),
```

Comment:
this seems to present the best format, but is this standard to use?

https://developer.apple.com/documentation/foundation/locale/identifiertype/bcp47